docs/business-apps/get-started/set-up-sharepoint-site-lists-libraries.md

```diff
@@ -1,20 +1,25 @@
 ---
 title: Set up your SharePoint site with lists and libraries
 description: Set up your SharePoint site with lists and libraries
-ms.date: 12/12/2018
+ms.date: 3/11/2020
 ms.prod: sharepoint
 localization_priority: Priority
 ---
 
 # Set up your SharePoint site with lists and libraries
-
 To successfully complete the *Getting Started* tutorials, set up your SharePoint site with the following items.
 
-## Projects list
+## Microsoft 365 tenant
+To use Power Automate and build flows, you need an Office 365 tenant.
```

Comment:
Microsoft 365 tenant in lieu of branding change afoot in company to swap Office 365 to Microsoft 365 under Project Snowbird.